debian/prerm

```diff
@@ -2,3 +2,9 @@
 systemctl stop wildfly || true
 systemctl disable wildfly || true
 
+# remove postgresql dependency on wildfly.service
+. "/usr/share/$DPKG_MAINTSCRIPT_PACKAGE/helper.sh"
+remove_entry_from_service "postgresql.service" "Requires" "wildfly.service"
+remove_entry_from_service "postgresql.service" "Before" "wildfly.service"
+
+systemctl daemon-reload
```

Comment:
Allgemein ist empfohlen, Dateien mit einem newline zu beenden.
https://thoughtbot.com/blog/no-newline-at-end-of-file